content/docs/alerting/configuration.md

```diff
@@ -454,6 +454,23 @@ channel: <tmpl_string>
 [ text: <tmpl_string> | default = '{{ template "slack.default.text" . }}' ]
 [ fallback: <tmpl_string> | default = '{{ template "slack.default.fallback" . }}' ]
 
+# An optional flag indicating whether the value(s) of all fields defined
+# are short enough to be displayed side-by-side with any other value(s).
+[ short_fields: <boolean> ]
+
+# An optional list of key/value pairs to provide further details about the incident.
+# The key/value pairs contained within it will be displayed in a table inside the
+# message attachment.
+[ fields:
```

Comment:
This is not the structure that is defined in the code.

Reply:
Not quite sure what you mean here. The way the code reads is a little misleading since it looks like it's looping over a map at first glance [[1](https://github.com/prometheus/alertmanager/blob/master/notify/impl.go#L679-L690)], but the structure is definitely a list of key/value pairs albeit very specific, i.e., include both both the title and value for the field. Thus, do you mean that the description in the preceding comment of the structure is inaccurate and needs to be more specific?

I've tested and used the following configuration for defining two different fields:

```yaml
fields:
- title: 'severity'
  value: '{{ .GroupLabels.severity }}'
- title: 'status'
  value: '{{ .Status }}'
```

Reply:
The code allows for any field to be supplied, not just title and value.

Reply:
Any number of fields can be defined, but it needs both the `title` [[1](https://github.com/prometheus/alertmanager/blob/master/notify/impl.go#L684)] and `value` [[2](https://github.com/prometheus/alertmanager/blob/master/notify/impl.go#L685)] defined for each field and then inherits the `short` parameter as a global setting for all fields [[3](https://github.com/prometheus/alertmanager/blob/master/notify/impl.go#L686), [4](https://github.com/prometheus/alertmanager/blob/master/config/notifiers.go#L247)]. This is the required structure for fields as per the Slack documentation [[5](https://api.slack.com/docs/message-attachments#fields)].

Reply:
That is still not what is implemented in the alertmanager. I think you need to fix the code.